src/lib/entities/content-type.ts

```diff
@@ -304,9 +304,27 @@ class EditorInterfaces {
     }
   }
 
-  // @ts-ignore
   deleteEditorLayoutFieldGroup (fieldGroupId: string) {
-    // TODO: delete field group
+    const fieldGroup = findFieldGroup(this._editorLayout, fieldGroupId)
+    if (!fieldGroup) {
+      return
+    }
+
+    const parentPath = fieldGroup.path.slice(0, -1)
+    const groupIndex = fieldGroup.path[fieldGroup.path.length - 1] as number
```

Comment:
Do we have a recent enough TypeScript version that we could use `fieldGroup.path.at(-1)`? ([MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at))

Reply:
Yes, we do 👍🏼